pkg/kubelet/kubelet_pods.go

```diff
@@ -1099,35 +1099,28 @@ func (kl *Kubelet) podKiller() {
 	killing := sets.NewString()
 	// guard for the killing set
 	lock := sync.Mutex{}
-	for {
-		select {
```

Comment:
`select` removed -> indentation changed. No code changes except for `range` in loop.